### ISSUE TITLE
Auto Answer still not working for fishtopia

### ISSUE THREAD

**Issue by eyeet535** [TheLazySquid/GimkitCheat#45]
Hey i know you said you fixed it but it still doesnt seem to be working

**Comment by TheLazySquid** [comment]
Everything works fine for me. Did you update the script? It should be at version 0.4.0 now. If this persists, please send information about what browser you are using and any errors in the console.

**Comment by eyeet535** [comment]
> Everything works fine for me. Did you update the script? It should be at version 0.4.0 now. If this persists, please send information about what browser you are using and any errors in the console.
my script is on 0.4.0 and im on brave browser

**Comment by eyeet535** [comment]
how do i open the console?

**Comment by eyeet535** [comment]
![1DDA81C0-E2D4-43C9-89BF-0CFB4185BEB2](https://github.com/TheLazySquid/GimkitCheat/assets/70739136/1f5a2255-78c1-4468-b144-e18973517b0c)

**Comment by TheLazySquid** [comment]
I just tried this out on Brave. Even with the instant inject mode experiment enabled, the userscript always loads in after the script it needs to intercept. I don't really have any idea of how to fix something like this. I'll keep you posted if I come up with something, but until then use a browser such as edge, chrome or firefox.

**Comment by TheLazySquid** [comment]
My bad, I completely forgot- on chrome or edge, you usually need to enable instant inject mode on tampermonkey. To do this, do the following:
Tampermonkey dashboard > Settings, set Config Mode to advanced
Scroll down to near the bottom, in the experimental section set inject mode to instant.

I'm going to try to make stuff work with default settings over the next few days, but no guarantees about whether its possible or not.

**Comment by TheLazySquid** [comment]
For now, I've added a warning if something goes wrong when loading the script. At least this way you can tell whether or not it's going to break before loading into the game.

**Comment by TheLazySquid** [comment]
Oh- I was way overthinking this. Even if we can't intercept the code before it loads, reimporting it with some changes will override the original code and hopefully make everything work. I've pushed a change, so this (in theory) works on all browsers with all inject modes. If this issue still persists please let me know.

**Comment by eyeet535** [comment]
it did fix but its broken again

**Comment by TheLazySquid** [comment]
It definitely broke again, although this time it was my fault. I was a bit lazy with setting up better injection for scripts, so the names were all hardcoded and when Gimkit pushed an update and the file names got changed the script couldn't figure out where to inject. Should all be working now, though.

**Comment by WGrav01** [comment]
Doesn't seem to be working on Firefox 123.0
![image](https://github.com/TheLazySquid/GimkitCheat/assets/63617338/5e699eb4-704a-484a-8022-47443873bb0b)

**Comment by TheLazySquid** [comment]
Those warnings are Gimkit's fault, not this script. As long as everything works you can safely ignore them.

**Comment by WGrav01** [comment]
Yes, but the script doesn’t work, while also spamming that warning.

**Comment by TheLazySquid** [comment]
I'm also using Firefox 123 and everything works fine for me; double check that the script is on the most up-to-date version (0.4.5) and if stuff still doesn't work tell me what gamemode you're trying to use it in.

**Comment by WGrav01** [comment]
Currently using 0.4.5, testing it out on Fishtopia as a homework assignment.

**Comment by The-Arx** [comment]
> I don't have any good way to test this, due to not having a teacher account to make assignments with :(

You can make a teacher account, there is no verification.
